README.md

```diff
@@ -123,6 +123,34 @@ Sort key: shard_id
 
 <img width="727" alt="screen shot 2017-11-22 at 7 59 36 pm" src="https://user-images.githubusercontent.com/739782/33158557-b90e4228-cfbf-11e7-9a99-73b56a446f5f.png">
 
+### Postgres Checkpoint
+
+The Postgres checkpoint requires Table Name, App Name, Stream Name and ConnectionString:
+
+```go
+import checkpoint "github.com/harlow/kinesis-consumer/checkpoint/postgres"
+
+// postgres checkpoint
+ck, err := checkpoint.New(app, table, connStr)
+if err != nil {
+    log.Fatalf("new checkpoint error: %v", err)
+}
+
+```
+
+To leverage the Postgres checkpoint we'll also need to create a table:
+
+```sql
+CREATE TABLE kinesis_consumer (
+	namespace text NOT NULL,
+	shard_id text NOT NULL,
+	sequence_number numeric NOT NULL,
```

Comment:
@imaramos almost there! have you tried running the code locally? I'm not sure about this `numeric` field. I think it might want to be text also.

```
{
  "SequenceNumber": "21269319989653637946712965403778482177",
  "ShardId": "shardId-000000000001"
}
```

Reply:
@harlow I have run it locally, and it worked. I think numeric should work just fine because its quite large https://www.postgresql.org/docs/9.1/static/datatype-numeric.html, also the sequence number shouldnt be larger than 128 digits (https://docs.aws.amazon.com/kinesis/latest/APIReference/API_SequenceNumberRange.html ) right? Didnt seem right  to me assign it text type when even the column name says that is a "number", but I understand your concerns, do you think to be sure, use like text?